src/components/screens/NewsItem.js

```diff
@@ -42,7 +42,7 @@ export const NewsItem = ({ data, navigation }) => {
     navigation.navigate({
       routeName: 'Web',
       params: {
-        title,
+        title: 'Nachrichte',
```

Comment:
'Nachricht' ⚠️

Reply:
"Please remove the variable, that is not used anymore" , do you mean const title ? I was getting a issues on when I removed it . 
"And please check the other detail screens, if they are all singular", I did they are all singular and Touren is always plural , right ?

Reply:
But then a single Tour needs to have "Tour" doesn't it?

Reply:
They are all singular now !